src/osd/osd_perf_counters.cc

```diff
@@ -338,6 +338,16 @@ PerfCounters *build_osd_logger(CephContext *cct) {
       l_osd_scrub_reservation_dur_hist, "scrub_resrv_repnum_vs_duration",
       rsrv_hist_x_axis_config, rsrv_hist_y_axis_config, "Histogram of scrub replicas reservation duration");
 
+  // mclock QoS queue op counter
```

Comment:
General comment about counter descriptions:

The counter description could just read "osd op_scheduler_class::immediate type op count" for example. The same applies to other counter descriptions. Remove the word "handle" from the descriptions.